README.md

```diff
@@ -44,16 +45,16 @@ The following flags can be used for each build:
 
 Flag | Description
 ---- | -----------
-`-tag` | The Vim release.  It must match the tags on Vim's [releases page](https://github.com/vim/vim/releases).
-`-name` | The name to use for the binary's symlink.  If omitted, the name will default to `vim-$TAG`.
+`-vim` | The Vim variant.  Either `vim` (default) or `nvim`.
```

Comment:
Actually, like `-tag neovim/neovim:v0.1.7` so it could be any git repo.  Like a fork `-tag tweekmonster/neovim:wip-branch` or something.